packages/tasks/src/model-libraries.ts

```diff
@@ -710,6 +710,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: true,
 		countDownloads: `path:"model_config.json"`,
 	},
+	paddleocr: {
```

Comment:
let's open PRs to add the tag: https://huggingface.co/models?other=paddleocr

Reply:
cc: @Wauplin - how should we tag here, the tag itself is `PaddleOCR` (case sensitive) but the tag/ key in the PR is always case insensitive no? (a bit confused with the casing)

because https://huggingface.co/models?other=PaddleOCR has a lot more models

whilst, https://huggingface.co/models?other=paddleocr has very few